### PR TITLE
Updated de_de

### PR DESCRIPTION
**Pull request by ichbineinNerd**
changed "mini" (well, mini) to "winzig" (tiny) and "keine weiche Beleuchtung" (no smooth lighting) to "normale Beleuchtung" (normal lighting)

### CODE REVIEW

**Comment by ichbineinNerd** [comment]
oh right and also added myself to the contributors

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 1]
```diff
@@ -88,13 +89,13 @@ options.framerateLimit.tooltip.5=Die Bildrate grenzt die FPS ein, sogar wenn
 options.framerateLimit.tooltip.6=die Begrenzung nicht erreicht ist.
 of.options.framerateLimit.vsync=V-Sync
 
-of.options.AO_LEVEL=Schattenhelligkeit
-of.options.AO_LEVEL.tooltip.1=Schattenhelligkeit
-of.options.AO_LEVEL.tooltip.2=  Aus - Keine Schatten
+of.options.AO_LEVEL=Weiche Beleuchtungs-level
+of.options.AO_LEVEL.tooltip.1=Weiche Beleuchtungs-level
+of.options.AO_LEVEL.tooltip.2=  AUS - Keine Schatten
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: AUS --> Aus

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 2]
```diff
@@ -154,11 +155,11 @@ of.options.CHUNK_LOADING.tooltip.8=die FPS erhöhen, indem es einen zweiten CPU-
...
 of.options.chunkLoading.multiCore=Multi-Core
 
-of.options.shaders=Shader-Effekte ...
-of.options.shadersTitle=Shader-Effekte
+of.options.shaders=Shader...
+of.options.shadersTitle=Shader
 
-of.options.shaders.packNone=Aus
-of.options.shaders.packDefault=(Interner Shader-Effekt)
+of.options.shaders.packNone=AUS
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: AUS --> Aus

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 3]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
-of.options.shaders.NORMAL_MAP.tooltip.7=Die normalen Karten werden vom aktivierten
+of.options.shaders.NORMAL_MAP.tooltip.7=Die Normalkarten werden vom aktivierten
 of.options.shaders.NORMAL_MAP.tooltip.8=Ressourcenpaket zur Verfügung gestellt.
 
-of.options.shaders.SPECULAR_MAP=Spiegelnde Karten
-of.options.shaders.SPECULAR_MAP.tooltip.1=Spiegelnde Karten
-of.options.shaders.SPECULAR_MAP.tooltip.2=  An - (Standard) Spiegelnde Karten aktiviert
-of.options.shaders.SPECULAR_MAP.tooltip.3=  Aus - Spiegelnde Karten deaktiviert
-of.options.shaders.SPECULAR_MAP.tooltip.4=Spiegelnde Karten können von Shader-Paketen verwendet
+of.options.shaders.SPECULAR_MAP=Reflektionsungskarten
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 4]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
+of.options.shaders.SPECULAR_MAP.tooltip.1=Reflektionsungskarten
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 5]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
+of.options.shaders.SPECULAR_MAP.tooltip.2=  An - (Standard) Reflektionsungskarten aktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 6]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
+of.options.shaders.SPECULAR_MAP.tooltip.3=  Aus - Reflektionsungskarten deaktiviert
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 7]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
+of.options.shaders.SPECULAR_MAP.tooltip.4=Reflektionsungskarten können von Shader-Paketen verwendet
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 8]
```diff
@@ -169,23 +170,23 @@ of.options.shaders.ANTIALIASING.tooltip.5=Linien und harte Farbübergänge glät
...
 of.options.shaders.SPECULAR_MAP.tooltip.5=werden, um spezielle Reflexionseffekte zu erzeugen.
-of.options.shaders.SPECULAR_MAP.tooltip.6=Die spiegelnden Karten werden vom aktivierten
+of.options.shaders.SPECULAR_MAP.tooltip.6=Die Reflektionsungskarten werden vom aktivierten
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Reflektionskarten ?

**Comment by TheHADILP** [comment]
Can you please split these comments into different PRs:
de_de
en_us
CTM-stuff

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 9]
```diff
@@ -76,7 +77,7 @@ options.renderDistance.tooltip.7=  64 Absurd - 1024m, braucht 3GB RAM zugeteilt
 options.renderDistance.tooltip.8=Werte über 16 funktionieren nur im Einzelspielermodus.
 
 options.ao.tooltip.1=Weiche Beleuchtung
-options.ao.tooltip.2=  Aus - Keine weiche Beleuchtung (schnell)
+options.ao.tooltip.2=  Aus - normale Beleuchtung (schnell)
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Normale (Uppercase)

File: OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang [hunk 10]
```diff
@@ -88,13 +89,13 @@ options.framerateLimit.tooltip.5=Die Bildrate grenzt die FPS ein, sogar wenn
...
-of.options.AO_LEVEL.tooltip.1=Schattenhelligkeit
-of.options.AO_LEVEL.tooltip.2=  Aus - Keine Schatten
+of.options.AO_LEVEL=Weiche Beleuchtungs-level
+of.options.AO_LEVEL.tooltip.1=Weiche Beleuchtungs-level
+of.options.AO_LEVEL.tooltip.2=  AUS - Keine Schatten
 of.options.AO_LEVEL.tooltip.3=  50%% - Helle Schatten
 of.options.AO_LEVEL.tooltip.4=  100%% - Dunkle Schatten
 
-options.viewBobbing.tooltip.1=Gehbewegung
+options.viewBobbing.tooltip.1=realistischere Gehbewegung
```
Comment (TheHADILP) [inline OptiFineDoc/assets/minecraft/optifine/lang/de_de.lang]: Realistischere (Uppercase)

**Comment by ichbineinNerd** [comment]
Sorry, I'm still kinda new to github, how do I split this to multiple PRs?
oh btw in the en_us, off on it's own is always spelled OFF

**Comment by ichbineinNerd** [comment]
I'm gonna close this if you're ok with it and try to make a new one just for de_de

**Comment by TheHADILP** [comment]
Yeah, no problem.
And yes, just make three new PRs for the different topics.
I know, in en_us ON and OFF are spelled in capital letters, but so far in de_de we have "An" "Aus" for all options.

**Pull request closed without merging**